notebooks/interactive_viz_demo.ipynb

```
@@ -0,0 +1,242 @@
+    "Launch this notebook with: `pixi run jupyter lab`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "a93y9vjvvzs",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
```

Ruff (0.15.10)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)